clinica/iotools/converters/adni_to_bids/adni_modalities/adni_pib_pet.py

```diff
@@ -62,6 +62,7 @@ def convert_adni_pib_pet(
     paths_to_bids(
         images, destination_dir, "pib", mod_to_update=mod_to_update, n_procs=n_procs
     )
+    paths_to_bids(images, destination_dir, "pib", mod_to_update=mod_to_update)
```

Comment:
This is a duplicate (see line just above):

```suggestion
```

Reply:
Noted